services/pdf-ms/helpers/objects.ts

```diff
@@ -0,0 +1,14 @@
+export function getPropertyFromDottedString(obj: Record<string, any>, dottedString: string): any {
+  const strArr = dottedString.split('.');
```

Comment:
Please use pronounceable variable names - strArr?!
Array of what sort of strings?